src/components/AttentionBox/__stories__/attentionBox.stories.mdx

```diff
@@ -0,0 +1,162 @@
+      <AttentionBox text="Attention box in compact mode" onClose compact componentClassName="monday-style-attention-box_box" />
+    </div>
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component: <AttentionBox title="Attention box title" text="Studies show that 100% of people who celebrate birthdays, will eventually die." componentClassName="monday-style-attention-box_box"/>,
+        description: "Provide a short and brief title, and explanation related to the title."
+      },
+      negative: {
+        component: <AttentionBox title="Attention box title" />,
+        description: "Don’t use only a title to explain something."
+      }
+    },
+    {
+      positive: {
+        component:
+          <div>
+            <span className="monday-style-attention-box_title">Inbox</span>
+            <span className="monday-style-attention-box_text" >Catch up on updates from all your boards.</span>
+            <AttentionBox text={<span className="monday-style-attention-box_info">Get your monday.com notifications<Link>Learn More</Link></span>} componentClassName="monday-style-attention-box_att-box"  />
+          </div>,
+        description: "Use when you are speaking directly to a piece of content and the notification can be positioned close to the content."
+      },
+      negative: {
+        component:
+          <div>
+            <span className="monday-style-attention-box_title">Inbox</span>
+            <span className="monday-style-attention-box_text" >Catch up on updates from all your boards.</span>
+            <AttentionBox text={<span className="monday-style-attention-box_info">7 days left on your free trial <Link>Learn More</Link></span>} compact componentClassName="monday-style-attention-box_att-box" />
+          </div>,
+        description: <>Don’t use when a message applies at a system level, does not apply to a specific piece of content, or needs to alert all users. On those cases, use <Link href="?path=/docs/components-alertbanner--overview" withoutSpacing>Alert banner.</Link></>
+      }
+    }
+  ]}
+/>
+
+## Use cases and examples
+### Natural attention box
+Provides additional information about an action or section.
+<Canvas>
+  <Story name="Natural attention box">
+    <div className="monday-style-attention-box_wrapper">
+      <span className="monday-style-attention-box_title">Cross-Account Copier</span>
+      <span className="monday-style-attention-box_text" >Copy boards and dashboards to another account</span>
+      <AttentionBox 
+        text={<span className="monday-style-attention-box_inline-wrapper"><Icon icon={Info} iconSize="18" />First, move the content you want to copy into folder. Only main boards and dashboards can be copied.<Link href="#" size={Link.sizes.SMALL} withoutSpacing>Learn more.</Link></span>}
```

Comment:
The attention box text should contain only string type text (not an element).
If the reason for the span wrapper is a difference between the design of attention box in figma and the default attention box appearance, please talk with me about it.

Reply:
for inline appearance please use the  compact prop

Reply:
Please also modify the icon of the attention box by using the icon and iconType props.If it's does not look the same as in the design please talk with me :)

Reply:
Without 'span' with class inside it does not look like in design, the 'icon' is not center.